pyanaconda/modules/network/initialization.py

```diff
@@ -111,6 +111,10 @@ def _run(self, nm_client):
                 log.warning("%s: --device %s not found", self.name, network_data.device)
                 continue
 
+            if is_nbft_device(device_name):
+                log.debug("Ignoring nBFT device %s", device_name)
+                continue
+
```

Comment:
I'm missing code coverage of this code.

Reply:
We don't have the code around covered at all, I am not able to add it today. It would involve quite some thinking and mocking.